web/src/components/BarcodeChart.vue

```diff
@@ -1,9 +1,11 @@
 <template >
   <div :id="identifier" class="barcodeChart">
     <component :is="'style'" type="text/css">
-      .marked {
-      fill: var(--markedbg);
-      }
+      <template v-for="item in activeSelections">
```

Comment:
Instead of adding a rule for every active selection, is it an idea to add a `.hidden` class to the unactive selections?

Reply:
This is definitely possible, however then the styling would no longer be completely controlled by the generated css. The other issue is that if a `.hidden` tag would be added programmatically then we would have to check for each span in the document if it has any active selections left before adding the tag, and this would more computationally then letting the css hanlde it.

Reply:
We'll leave it like this then.